samples/GalleryApp/Pages/Skills.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "Name": "Vision",
+    "Skills": [
+      {
+        "Name": "Skeletal Detector",
+        "Description": "A set of sample apps that can detect and identify the poses of the bodies of individuals in a video or image.",
```

Comment:
>Description [](start = 9, length = 11)

if you can avoid copying the Name and Description text here and instead rely on the ISkillDexcription.Description property of each skill (simply instantiate it and retrieve it/cache it at UI population time) you would win some time / avoid mismatch